x-pack/plugin/eql/qa/security/src/javaRestTest/java/org/elasticsearch/xpack/eql/AsyncEqlSecurityIT.java

```diff
@@ -89,8 +89,8 @@ private void testCase(String user, String other) throws Exception {
         }
         ResponseException exc = expectThrows(ResponseException.class,
             () -> submitAsyncEqlSearch("index-" + other, "*", TimeValue.timeValueSeconds(10), user));
-        assertThat(exc.getResponse().getStatusLine().getStatusCode(), equalTo(403));
-        assertThat(exc.getMessage(), containsString("unauthorized"));
+        assertThat(exc.getResponse().getStatusLine().getStatusCode(), equalTo(404));
+        //assertThat(exc.getMessage(), containsString("unauthorized"));
```

Comment:
@imotov not sure why this fails now... Essentially by changing the index option `ignoreUnavailable` from `false` to `true` the 403/unauthorized became a 404...

Reply:
I think it is because we ignore unavailable indices and ended up with no indices at all, which leads to 404.

Reply:
And previously we did not ignore them but because we couldn't get access to them to verify their existence we got the 403...Make sense.